user_guide_src/source/outgoing/response/031.php

```diff
@@ -2,15 +2,11 @@
 
 // Go back to the previous page.
 return redirect()->back();
-
 // Keep the old input values upon redirect so they can be used by the `old()` function.
 return redirect()->back()->withInput();
-
 // Set a flash message.
 return redirect()->back()->with('foo', 'message');
-
 // Copies all cookies from global response instance.
 return redirect()->back()->withCookies();
-
```

Comment:
It seems difficult to read. Why don't you remove `return`?
![Screnshot 2023-07-22 17 16 28](https://github.com/codeigniter4/CodeIgniter4/assets/87955/95590e03-0764-4492-91c5-3f5696655751)

Reply:
I also liked the previous version better.